contentcuration/contentcuration/frontend/channelList/views/Channel/components/StudioChannelCard.vue

```diff
@@ -0,0 +1,431 @@
+            // this component is used on the catalog search
+            // page => do not lose search query
+            ...this.$route.query,
+            last: this.$route.name,
+          },
+          params: {
+            channelId: channel.id,
+            tab: 'edit',
+          },
+        });
+      },
+      handleDelete() {
+        if (!this.canEdit) {
+          const currentUserId = this.$store.state.session.currentUser.id;
+          this.removeViewer({ channelId: this.selectedChannel.id, userId: currentUserId }).then(
+            () => {
+              this.deleteDialog = false;
+              this.$store.dispatch('showSnackbarSimple', this.$tr('channelRemovedSnackbar'));
+            },
+          );
+        } else {
+          this.deleteChannel(this.selectedChannel.id).then(() => {
+            this.deleteDialog = false;
+            this.$store.dispatch('showSnackbarSimple', this.$tr('channelDeletedSnackbar'));
+          });
+        }
+      },
+      trackTokenCopy() {
+        this.$analytics.trackAction('channel_list', 'Copy token', {
+          eventLabel: this.selectedChannel.primary_token,
+        });
+      },
+      thumbnailSrc(channel) {
+        return channel.thumbnail_encoding && channel.thumbnail_encoding.base64
+          ? channel.thumbnail_encoding.base64
+          : channel.thumbnail_url;
+      },
+      hasUnpublishedChanges(channel) {
+        return !channel.last_published || channel.modified > channel.last_published;
+      },
+      getPublishStatus(channel) {
+        return channel.last_published
+          ? this.$tr('lastPublished', {
+            last_published: this.$formatRelative(channel.last_published, {
+              now: new Date(),
+            }),
+          })
+          : this.$tr('unpublishedText');
+      },
+    },
+    $trs: {
+      resourceCount: '{count, plural,\n =1 {# resource}\n other {# resources}}',
+      unpublishedText: 'Unpublished',
+      lastPublished: 'Published {last_published}',
+      lastUpdated: 'Updated {updated}',
+      details: 'Details',
+      viewContent: 'View channel on Kolibri',
+      goToWebsite: 'Go to source website',
+      editChannel: 'Edit channel details',
+      copyToken: 'Copy channel token',
+      deleteChannel: 'Delete channel',
+      deleteTitle: 'Delete this channel',
+      removeChannel: 'Remove from channel list',
+      removeBtn: 'Remove',
+      removeTitle: 'Remove from channel list',
+      deletePrompt: 'This channel will be permanently deleted. This cannot be undone.',
+      removePrompt:
+        'You have view-only access to this channel. Confirm that you want to remove it from your list of channels.',
+      channelDeletedSnackbar: 'Channel deleted',
+      channelRemovedSnackbar: 'Channel removed',
+      channelLanguageNotSetIndicator: 'No language set',
+      cancel: 'Cancel',
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .channel-card {
+    width: 100%;
+  }
+
+  .cards-below-title {
```

Comment:
It'd be meaningful to remove `.cards-` prefix from this and other classes - since now we're inside the card component.